misc changes to deva alt layout

lrvideckis opened the pull request:
- remove keypad key in favor of circular swipes. This allows a larger backspace key
- move around some symbols in a way which I've found more convenient:
   - ऽ key is now a tap because i use it frequently
   - ् key moved to center to be convenient to tap with both thumbs as i use this frequently
   - । key is now a swipe (previously it was a tap); i use this symbol very seldom

lrvideckis commented:
![IMG_20250118_145009](https://github.com/user-attachments/assets/e30a388c-c040-4839-be3d-1653dea4f8be)

Julow commented on srcs/layouts/deva_alt.xml:
```diff
@@ -3,28 +3,27 @@
-    <key              width="0.9" key0="ट" key1="ठ" key2="ढ" key3="ण"  key4="ड"/>
-    <key              width="0.9" key0="त" key1="थ" key2="ध" key3="न"  key4="द"/>
-    <key              width="0.9" key0="प" key1="फ" key2="भ" key3="म"  key4="ब"/>
+    <key              width="0.9" key0="ट" key1="ठ" key2="ढ" key3="ण"  key4="ड" anticircle="७"/>
```
The anticircle might be hard to discover. Perhaps you can add `indication="७"` on every keys with a gesture to make them visible to users ?

Julow reviewed (commented):
Thanks for maintaining the layout!

lrvideckis commented:
one more change: I removed the `*` swipe. I originally added this because when I text my indian friends, and I make a typo, I can quickly correct myself by fixing the typo and typing `*`

But since I added `*`, whatsapp added the feature to edit messages. So I just do this now, and I no longer use `*`

lrvideckis commented:
![IMG_20250126_181824](https://github.com/user-attachments/assets/8b13b138-b62f-427d-bbe1-da276ae29fe4)

Julow reviewed (approved):
Awesome, thanks!
